eq-author/src/apollo/createApolloErrorLink.test.js

```diff
@@ -1,27 +1,83 @@
+import uuid from "uuid";
+import jwt from "jsonwebtoken";
 import { errorHandler } from "./createApolloErrorLink";
+import { sendSentryError, setSentryTag, setSentryUser } from "./sentryUtils";
 
-const dispatch = jest.fn();
-
-const createMockStore = () => ({
-  dispatch,
-});
+jest.mock("./sentryUtils");
 
 describe("createErrorLink", () => {
+  let dispatch;
+
+  const createAccessToken = (token, signingKey = uuid.v4()) => {
+    return jwt.sign(token, signingKey);
+  };
+
+  const createMockStore = () => ({
+    dispatch,
+  });
+
+  beforeAll(() => {
+    /* eslint-disable-next-line camelcase */
+    const tokenData = { user_id: "Test User", email: "test.user@email.com" };
+    const token = createAccessToken(tokenData);
+    localStorage.setItem("accessToken", token);
+
+    dispatch = jest.fn();
+  });
+
+  let networkError = {
```

Comment:
Can you define these in beforeEach so that you are guaranteed they are not shared between tests.